src/components/CheckIn.js

```diff
@@ -0,0 +1,12 @@
+import { useParams, useHistory } from "react-router";
+import { useEffect } from "react";
+
+export default function CheckIn({ updateTableId }) {
+  let history = useHistory();
+  const { tableId } = useParams();
+  useEffect(() => {
+    updateTableId(tableId);
+    history.push("/menu");
+  });
+  return <></>;
```

Comment:
Interseting solution! If you don't want to render anything, you can as well return null